Scala/syntax_test_scala.scala

```diff
@@ -306,6 +306,10 @@ type Foo = Bar[A] forSome { type A }
 // <- comment.block.documentation.scala
 */
 
+  /**/0xff
+//^^^^ comment.block.empty.scala
+//    ^^^^ - thereafter, not a comment 
```

Comment:
@jpz it sounds like there is some confusion about how the tests and scope selectors work. If the line starts with a comment, and contains `^` or `<-` then it is a test assertion line (described at http://www.sublimetext.com/docs/3/syntax.html#testing). The rest of the line then dictates the scope selector. Because the scope selector is `- comment` and not just `comment`, it means, assert that the scope does *not* match `comment`. I've never really found any good documentation on scope selectors, but an answer I wrote on SO may be useful: http://stackoverflow.com/questions/37406338/sublime-text-modifying-tmtheme-file/37407588#37407588